hw/opentitan/ot_keymgr.c

```diff
@@ -796,7 +892,34 @@ static bool ot_keymgr_main_fsm_tick(OtKeyMgrState *s)
         break;
     }
 
-    /* @todo: update ERR_CODE, OP_STATUS & CFG_REGWEN based on FSM changes */
+    /* update the last requested operation status */
+    bool invalid_op = (bool)(s->regs[R_ERR_CODE] & R_ERR_CODE_INVALID_OP_MASK);
+    bool op_done =
+        s->op_state.op_req ? s->op_state.op_ack : (init || invalid_op);
+    if (op_done) {
```

Comment:
I think we're missing an `op_done` interrupt here
https://opentitan.org/book/hw/ip/keymgr/doc/interfaces.html#interrupts